### DISCUSSION TITLE
I am trying to send transaction (send Ether using trustwallet after connection) Not working

### DISCUSSION THREAD

**Discussion by cryptonexum** [wevm/wagmi#2745] [General]
wallet is connected  but No popup for transaction is coming on trustwallet. however its working fine with metamask perfectly.

here is my  code

`import {sendTransaction} from "@wagmi/core";
 let tx = {
    to: '0x0000000000000000000000000000000000000000',
    gasLimit: '300000',
    nonce: 1,
    chainId: 1,
    value: "1"
  };
  const result = await sendTransaction(tx);
  console.log(result);
`

On debug I got this

![WhatsApp Image 2023-07-16 at 12 29 14 PM](https://github.com/wagmi-dev/wagmi/assets/139538354/70d68c44-5366-4f7f-9ff4-b42c08211f35)

![WhatsApp Image 2023-07-16 at 12 29 14 PM](https://github.com/wagmi-dev/wagmi/assets/139538354/86d6acae-fd77-4ac9-8595-cce423cb24c4)

Please help

**Comment by meri-maki** [comment]
@cryptonexum add ```data: "0x0"``` to ```let tx = { to: '0x0000000000000000000000000000000000000000', gasLimit: '300000', nonce: 1, chainId: 1, value: "1" };```
see https://github.com/WalletConnect/web3modal/issues/1082
